fe/fe-core/src/main/java/org/apache/doris/datasource/es/EsUtil.java

```diff
@@ -307,6 +310,7 @@ private static Column parseEsField(String fieldName, ObjectNode fieldValue, List
         } else {
             type = Type.JSONB;
```

Comment:
STRING is better

Reply:
When there is no explicit type in mapping, it indicates this type is an `object` in Elasticsearch.